include/vkgcDefs.h

```diff
@@ -82,6 +82,7 @@
 //  %Version History
 //  | %Version | Change Description                                                                                    |
 //  | -------- | ----------------------------------------------------------------------------------------------------- |
+//  |     54.1 | Add subgroupSize to PipelineShaderOptions                                                             |
```

Comment:
This will conflict with https://github.com/GPUOpen-Drivers/llpc/pull/1922.  Just an FYI.